api/open_saves.proto

```diff
@@ -580,6 +586,16 @@ message CommitChunkedUploadRequest {
   Hint hint = 2;
 }
 
+message CommitChunkedUploadWithUpdateRecordRequest {
+  // session_id is the ID of a chunked upload session to commit.
+  string session_id = 1;
+
+  // Performance hints.
+  Hint hint = 2;
+
+  UpdateRecordRequest update_record = 3;
```

Comment:
This member variable also has a Hint as part of its definition. I suggest we either:
(1) Specify all three property (hint, storeKey, and record) as part of the request payload or,
(2) We remove the hint field from the CommitChunkedUploadWithUpdateRecordRequest struct to avoid redundancy